### ISSUE TITLE
Loro-wasm won't run on a non-node environment such as CloudFlare Workers

### ISSUE THREAD

**Issue by raymonddaikon** [loro-dev/loro#440]
Bindgen isn't available on non-node environments so a patch is needed to allow for `loro-wasm` to be used server-side. I was able to patch this in the JS output file `loro_wasm.js` by modifying it based on the note found [here](https://developers.cloudflare.com/workers/languages/rust/#javascript-plumbing-wasm-bindgen). Automerge had a similar issue with their wasm import that they fixed with this PR https://github.com/automerge/automerge-repo/pull/372, however I'm not sure if it's applicable here. 

```import * as wasm from "./loro_wasm_bg.wasm";
import * as imports from "./loro_wasm_bg.js";

if (wasm.__wbindgen_start) {
  imports.__wbg_set_wasm(wasm);
  wasm.__wbindgen_start();
} else {
  const wkmod = await import('./loro_wasm_bg.wasm')
  const instance = new WebAssembly.Instance(wkmod.default, {
    "./loro_wasm_bg.js": imports,
  });
  imports.__wbg_set_wasm(instance.exports);
}
export * from "./loro_wasm_bg.js";```

**Comment by zxch3n** [comment]
Thanks! I was not aware of that.

The mentioned automerge issue seems like another problem, though.

**Comment by zxch3n** [comment]
Is there any other environment you found that doesn't work besides Cloudflare Worker?

**Comment by raymonddaikon** [comment]
I'll have to do some more testing on other environments but I'm assuming it's mainly on these custom js runtimes. I believe the fix is adding some build targets, but I'm not familiar enough with rust to say.

**Comment by zxch3n** [comment]
The fix is published to v0.16.11
